coffeelint.json

```diff
@@ -37,7 +37,7 @@
         "value": "unix"
     },
     "max_line_length": {
-        "value": 80,
+        "value": 100,
```

Comment:
I just didn't want to waste time fixing line length for semantic analysis expected output.